python/tests.py

```diff
@@ -132,3 +133,10 @@ def test_has_key(self):
         config = Config(self._json_config)
         self.assertTrue("the_environment" in config)
         self.assertTrue("does_not_exist" not in config)
+
+    def test_key_as_index(self):
+        config = Config(self._json_config)
+        my_string = "thekey"
+        self.assertEqual(config.the_environment[my_string], "TheValue")
+
+unittest.main()
```

Comment:
Looks like I should remove unittest.main() as I was running python without the `-m` option that the rakefile calls it with.